### PR TITLE
[unit-tests-only] Add unit tests for AppBar component

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Adds unit  tests for `AppBar` component

## Related Issue
- Part of #5234

## Motivation and Context
- increase unit tests coverage

## How Has This Been Tested?
- test environment: local & CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+            false,
+            fileHandler.ext,
+            fileHandler.action
+          )
+        }
+      )
+    })
+
+    describe('when an item is selected', () => {
+      it('should show size info', () => {
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js:178]: you've tested that new but is not visible when items are not selected. can the counterpart go inside this describe?

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

File: packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+      name: page,
+      params: {
+        item: page === 'files-public-list' ? '6mfXfTtYHVxrlAu' : ''
+      },
+      meta: {
+        hideFilelistActions: false
+      }
+    }
+
+    describe('when items are not selected', () => {
```
Comment (individual-it) [inline packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js:103]:
```suggestion
    describe('when no items are selected', () => {
```

File: packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+
+    it('should not show "New" button and file menu list', () => {
+      const newButton = wrapper.find(elSelector.newFileButton)
+      const ocDrop = wrapper.find(elSelector.ocDrop)
+
+      expect(newButton.exists()).toBeFalsy()
+      expect(ocDrop.exists()).toBeFalsy()
+    })
+
+    describe('when items are not selected', () => {
```
Comment (individual-it) [inline packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js:215]:
```suggestion
    describe('when no items are selected', () => {
```

File: packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+        }
+      }
+
+      let wrapper
+      beforeEach(() => {
+        const store = createStore({ selected: [], currentFolder })
+        wrapper = getShallowWrapper(route, store)
+      })
+
+      describe('when items are not selected', () => {
```
Comment (individual-it) [inline packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.js:256]:
```suggestion
      describe('when no items are selected', () => {
```

**Review by individual-it — CHANGES_REQUESTED** [review]
do we need other tests for when one item is selected vs. multiple selected items?

**Review by individual-it — APPROVED** [review]

**Pull request merged**